packages/react/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@latticexyz/react",
+  "version": "1.32.0",
+  "description": "React tools for MUD client.",
+  "license": "MIT",
+  "source": "src/index.ts",
+  "main": "src/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/react"
+  },
+  "scripts": {
+    "prepare": "yarn build",
+    "link": "yarn link",
+    "docs": "rimraf API && typedoc src && find API -type f -name '*.md' -exec sed -E -i \"\" \"s/(#.*)(<.*>)/\\1/\" {} \\; && echo 'label: API' > API/index.yml",
+    "test": "tsc && jest",
+    "prepack": "mv package.json package.json.bak && jq \".main = \\\"dist/index.js\\\"\" package.json.bak > package.json ",
+    "postpack": "mv package.json.bak package.json || echo 'no package.json.bak'",
+    "build": "rimraf dist && rollup -c rollup.config.js",
+    "release": "npm publish || echo 'version already published'"
+  },
+  "devDependencies": {
+    "@latticexyz/recs": "^1.32.0",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "@rollup/plugin-typescript": "^8.3.1",
+    "@testing-library/react-hooks": "^8.0.1",
+    "@types/react": "^18.0.12",
+    "eslint-plugin-react": "^7.31.11",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "jest": "^29.3.1",
+    "mobx": "^6.4.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
is `react-dom` also required as peer dependency?

Reply:
nope! will remove